### DISCUSSION TITLE
Comments as Suggestions

### DISCUSSION THREAD

**Discussion by MrBrain295** [community/community#11431] [Copilot Conversations]
GitHub Copilot started working but it gave me comments as code. For example I commented `# find the greatest common factor of two numbers n and m` and it gave me `# gcf(n, m) = gcf(m, n mod m)
# 
# if n mod m = 0 then gcf(n, m) = m
# else gcf(n, m) = gcf(n mod m, m)
# 
# The greatest common factor of n and m is the largest number that divides both n and m.
# 
# Example:
# 
# gcf(12, 15) = 3
# 
# gcf(12, 15) = gcf(15, 12 mod 15)
# 
# gcf(12 mod 15, 15)
# 
# gcf(15 mod 12, 15)
# 
# gcf(15 mod 12, 15 mod 12)
# 
# gcf(15 mod 12, 15 mod 12) = 15
# 
# gcf(15 mod 12, 15 mod 12) = gcf(15 mod 12, 15 mod 12 mod 15)
# 
# gcf(15 mod 12, 15 mod 12 mod 15)
# 
# gcf(15 mod 12 mod 15, 15 mod 12)
# 
# gcf(15 mod 12, 15 mod 12 mod 15)
# 
# gcf(15 mod 12, 15 mod 12 mod 15) = gcf(15 mod 12, 15 mod 12 mod 15 mod 12)
# 
# gcf(15 mod 12, 15 mod 12 mod 15 mod 12)
# 
# gcf(15 mod 12, 15 mod 12 mod 15 mod 12) = gcf(15 mod 12, 15 mod 12 mod 15 mod 12 mod 15)
# 
# gcf(15 mod 12, 15 mod 12 mod 15 mod 12 mod 15)
# 
# gcf(15 mod 12, 15 mod 12 mod 15 mod 12 mod 15) = gcf(15 mod 12, 15 mod 12 mod 15 mod 12 mod 15 mod 12)
# 
# gcf(15 mod 12, 15 mod 12 mod 15 mod 12 mod 15 mod 12)
# 
# gcf(15 mod 12, 15 mod 12 mod 15 mod 12 mod 15 mod 12) = gcf(15 mod 12, 15 mod 12 mod 15 mod 12 mod 15 mod 12 mod 15)` sorry it's all in headings it's like that due to #s being Python comments.

**Comment by KTibow** [comment]
> sorry it's all in headings it's like that due to #s being Python comments.

You can make something a block of code by using backticks around it. If you don't get what I mean, try selecting the code, and press the button that looks like `< >`.

**Reply by MrBrain295** [reply]
That’s what I was doing but the #s were appearing as headings.

**Reply by KTibow** [reply]
```
That shouldn't happen.
# For context, I wrote this whole comment as a code block.

# None of the comments are appearing as headings.
# It looks like you used quote formatting, not code formatting.
```

**Reply by MrBrain295** [reply]
It is working for my first code block so I don’t know why it isn’t working for the 2nd code block.

**Reply by KTibow** [reply]
You need to use triple backticks, on their own line, and close them later.
